ontology/yaml/resources/subfields/subfields.yaml

```diff
@@ -66,6 +66,7 @@ component:
 descriptor:
   air: "Atmospheric air, either conditioned or unconditioned."
   aisle: "Passage between two rows of server racks."
+  window: "Transparent part of the facade"
```

Comment:
Please place in alphabetical order within the descriptor section

Reply:
...done